game/scripts/npc/heroes/huskar.txt

```diff
@@ -0,0 +1,10 @@
+"DOTAHeroes"
+{
+  //=================================================================================================================
+  // HERO: Huskar
+  //=================================================================================================================
+  "npc_dota_hero_huskar"
+  {
+    "Ability16"                                   "special_bonus_magic_resistance_80" // replaces life break talent         
```

Comment:
@DarkoniusXNG what about this talent instead? Bring it back to the good ol days of Huskar not takin magic damage :D

Reply:
Better than attack range, still debatable if better than pure spears. Huskar's main problem late game is being stunned and takin physical damage not so much magic dmg but it might be nice for him to be near-immune to magic dmg at late game situations.

Reply:
85% magic resistance total combined with base, so maybe it's not super duper op but just slightly op, like pure dmg spears.

Reply:
this is a way better direction, there is no chance 80% magic resistance gets merged. best bet is that its going to get halved

Reply:
40% magic resistance is better.

Reply:
> 40% magic resistance is better.

50 and we have a deal.